Broken on 4.03+trunk

```
/home/whitequark/.opam/4.03.0+trunk/build/ctypes.0.3.3/src/ctypes/type_info_stubs.c: In function ‘ctypes_read’:
/home/whitequark/.opam/4.03.0+trunk/build/ctypes.0.3.3/src/ctypes/type_info_stubs.c:51:44: error: ‘uint64’ undeclared (first use in this function)
    case Uint64_t: b = ctypes_copy_uint64(*(uint64 *)buf); break;
                                            ^
/home/whitequark/.opam/4.03.0+trunk/build/ctypes.0.3.3/src/ctypes/type_info_stubs.c:51:44: note: each undeclared identifier is reported only once for each function it appears in
/home/whitequark/.opam/4.03.0+trunk/build/ctypes.0.3.3/src/ctypes/type_info_stubs.c:51:52: error: expected expression before ‘)’ token
    case Uint64_t: b = ctypes_copy_uint64(*(uint64 *)buf); break;
                                                    ^
/home/whitequark/.opam/4.03.0+trunk/build/ctypes.0.3.3/src/ctypes/type_info_stubs.c: In function ‘ctypes_write’:
/home/whitequark/.opam/4.03.0+trunk/build/ctypes.0.3.3/src/ctypes/type_info_stubs.c:92:21: error: ‘uint64’ undeclared (first use in this function)
    case Uint64_t: *(uint64 *)buf = Uint64_val(v); break;
                     ^
/home/whitequark/.opam/4.03.0+trunk/build/ctypes.0.3.3/src/ctypes/type_info_stubs.c:92:29: error: expected expression before ‘)’ token
    case Uint64_t: *(uint64 *)buf = Uint64_val(v); break;
                             ^
make: *** [_build/src/ctypes/type_info_stubs.o] Error 2
```

--- a/src/ctypes/type_info_stubs.c
+++ b/src/ctypes/type_info_stubs.c
@@ -58,7 +58,7 @@ value ctypes_read(value prim_, value buffer_)
    case Uint8_t: b = ctypes_copy_uint8(*(uint8_t *)buf); break;
    case Uint16_t: b = ctypes_copy_uint16(*(uint16_t *)buf); break;
    case Uint32_t: b = ctypes_copy_uint32(*(uint32_t *)buf); break;
-   case Uint64_t: b = ctypes_copy_uint64(*(uint64 *)buf); break;
+   case Uint64_t: b = ctypes_copy_uint64(*(uint64_t *)buf); break;
    case Camlint: b = Val_int(*(intnat *)buf); break;
    case Nativeint: b = caml_copy_nativeint(*(intnat *)buf); break;
    case Float: b = caml_copy_double(*(float *)buf); break;

